lib/oli_web/templates/project/overview.html.eex

```diff
@@ -115,6 +118,61 @@
           <% end %>
         </div>
       </div>
+
+      <div class="row my-4">
+        <div class="col-12">
+          <h4>Delete this course</h4>
+          <button type="button" class="btn btn-danger" onclick="showDeleteModal()">Delete this Course</button>
+        </div>
+      </div>
+
     </div>
   </div>
 </div>
+<script>
+  const showDeleteModal = () => {
+    $('#delete-modal').modal('show');
+  }
+  $(function () {
+    const deleteTitleInput = document.querySelector('#delete-confirm-title');
+    if (deleteTitleInput) {
+      deleteTitleInput.addEventListener("input", function (e) {
+        const value = e.target.value;
+        console.log(value)
```

Comment:
console.log should be removed (ESLint would have caught this if this code was in a TypeScript file)

Reply:
deleted console.log